components/public-api/gitpod/v1/organization.proto

```diff
@@ -98,6 +99,9 @@ message UpdateOrganizationSettingsRequest {
 
   // pass empty string to reset to the installation default workspace image
   optional string default_workspace_image = 4;
+
+  // pass empty string to unset maximum_workspace_class
+  optional string maximum_workspace_class = 5;
```

Comment:
that's questionable, is there well defined order between classes? Classes are delivered from the workspace cluster, should be order defined there that the engine team does not break it later? What happens if the new class is added? cc @loujaybee 

alternatives is to allow enable/disable workspace classes